### ISSUE TITLE
State 'libvirt.keys' was not found in SLS 'libvirt.keys' for Salt 2016.11.0

### ISSUE THREAD

**Issue by haraldrudell** [saltstack-formulas/libvirt-formula#22]
```
          ID: libvirt.keys
    Function: libvirt.keys
        Name: libvirt_keys
      Result: False
     Comment: State 'libvirt.keys' was not found in SLS 'libvirt.keys'
              Reason: 'libvirt.keys' is not available.
 ```

because in keys.sls line 6 a comparison is made as text, where it needs to be numeric:
```
{% if salt['grains.get']('saltversion', '') < '2016.3.0' %}
```

Here is my grains data:
```
sudo salt c89 grains.items | grep --after-context=1 saltversion:
    saltversion:
        2016.11.0
```

**Comment by haraldrudell** [comment]
The saltversioninfo array can be used:

```
sudo salt c89 grains.items | grep --after-context=4 saltversioninfo:
    saltversioninfo:
        - 2016
        - 11
        - 0
        - 0
```
If there is not sort --version-sort like function

**Comment by ukretschmer** [comment]
This was already done: https://github.com/saltstack-formulas/libvirt-formula/pull/21

So simply checkout the latest version of the formula and you are fine with that.

**Comment by baby-gnu** [comment]
Hello.

I close this one since there is a newer one #29 and I proposed a fix for it as #34.

Regards.
